### PR TITLE
upgrade yourkit to latest

### PR DESCRIPTION
**Pull request by fatlazycat**
version YourKit Java Profiler 2016.02, build 29

### CODE REVIEW

File: Casks/yourkit-java-profiler.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  version '2016.02-b29'
+  sha256 'd65adc15bbdc030767d909e4ce78a26fbae462862665b3a0de87e88bc07a3a7f'
 
   url "https://www.yourkit.com/download/yjp-#{version}-mac.zip"
   name 'YourKit Java Profiler'
   homepage 'https://www.yourkit.com/overview/'
   license :commercial
 
-  app "YourKit_Java_Profiler_#{version.gsub('-', '_')}.app"
+  app "YourKit-Java-Profiler-#{version.split('-')[0]}.app"
```
Comment (vitorgalvao) [inline Casks/yourkit-java-profiler.rb]: Use `#{version.major_minor}` instead.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)? Please @mention me after you do, so I get an email to check back.

**Comment by fatlazycat** [comment]
@vitorgalvao squashed

**Comment by vitorgalvao** [comment]
Merged as 75d21af0a7c4f1dc3b89db1f534b5a79a69ac83b.

**Pull request closed without merging**